p4-16/psa/PSA.mdk

```diff
@@ -477,8 +477,7 @@ The list above is not intended to be exhaustive.
 All of the comments above apply to all types where numerical
 translation occurs between the controller and the data plane.
 Below is a complete list of such types planned for numerical
-translation by default with PSA, and have an annotation
-`p4runtime_translation` in the `psa.p4` include file:
+translation by default with PSA:
```

Comment:
Nit (editorial): Below is a complete list of numeric types translated by default in PSA:

Reply:
Changed.